jsonnet/thanos-querier.libsonnet

```diff
@@ -266,8 +266,27 @@ function(params)
 
     deployment+: {
       spec+: {
+        strategy+: {
+          // Apply HA conventions
+          rollingUpdate: {
+            maxUnavailable: '25%',
+          },
+        },
         template+: {
           spec+: {
+            affinity+: {
+              podAntiAffinity+: {
+                // Apply HA conventons
+                requiredDuringSchedulingIgnoredDuringExecution: [
```

Comment:
we need to remove the `preferredDuringSchedulingIgnoredDuringExecution` key.

Reply:
indeed, I missed the existing anti-affinity configuration